### DISCUSSION TITLE
Running synpress as a github action?

### DISCUSSION THREAD

**Discussion by avp1598** [Synthetixio/synpress#308] [Q&A] [answered]
Hi, I wanted to know if there is any way of running synpress as a github action on push similar to how cypress action works. 
I think it would be possible if synpress were a cypress plugin but I am not sure how to do it. 

Thanks!

**Comment by drptbl** [comment, marked as answer]
Hey @avp1598.

Yes, you can run synpress as github action. This is what we do on our repositories.

You can find an example here:
https://github.com/Synthetixio/synpress/blob/dev/.github/workflows/audit_and_lint.yml#L84
+
https://github.com/drptbl/synpress-setup-example/blob/master/.github/workflows/e2e.yml

Also a lot of informations in this issue:
https://github.com/Synthetixio/synpress/issues/498#issuecomment-1251504500
https://github.com/Synthetixio/synpress/issues/498#issuecomment-1252191083

Thanks,
Jakub.

**Reply by kwdikeman** [reply]
Link does not work.

**Reply by drptbl** [reply]
> Link does not work.

@kwdikeman link updated, thanks for letting me know 🤝.

**Reply by arielmachado-uy** [reply]
Hi, the link is broken again. Could you please update it? Thanks in advance

**Reply by akkikumar72** [reply]
Broken again. Please update it

**Reply by drptbl** [reply]
@arielmachado-uy @akkikumar72 updated.

**Reply by Mahmoud-Kandel** [reply]
Broken again

**Reply by steezeburger** [reply]
Can you update the link to current examples?
